Gemfile.lock

```diff
@@ -209,60 +209,60 @@ GEM
       sprockets-rails
       tilt
     semantic_range (2.3.0)
-    spring (2.1.0)
+    spring (2.1.1)
     spring-watcher-listen (2.0.1)
       listen (>= 2.7, < 4.0)
       spring (>= 1.2, < 3.0)
     sprockets (4.0.2)
       concurrent-ruby (~> 1.0)
       rack (> 1, < 3)
-    sprockets-rails (3.2.1)
+    sprockets-rails (3.2.2)
       actionpack (>= 4.0)
       activesupport (>= 4.0)
       sprockets (>= 3.0.0)
     thor (1.0.1)
     thread_safe (0.3.6)
     tilt (2.0.10)
-    tinymce-rails (5.4.1)
+    tinymce-rails (5.6.1)
       railties (>= 3.1.1)
-    trestle (0.9.3)
+    trestle (0.9.4)
       activemodel (>= 4.2.0)
       kaminari (>= 1.1.0)
       railties (>= 4.2.0)
       sprockets-rails (>= 2.0.0)
-    trestle-active_storage (3.0.0)
+    trestle-active_storage (3.0.1)
       rails (>= 5.2, < 7)
       trestle (~> 0.9.0, >= 0.9.3)
-    trestle-auth (0.4.0)
+    trestle-auth (0.4.1)
       bcrypt (~> 3.1.7)
-      trestle (~> 0.9.0)
+      trestle (~> 0.9.0, >= 0.9.3)
     trestle-tinymce (0.3.0)
       tinymce-rails (~> 5.1)
       trestle (~> 0.9.0)
     turbolinks (5.2.1)
       turbolinks-source (~> 5.2)
     turbolinks-source (5.2.0)
-    tzinfo (1.2.7)
+    tzinfo (1.2.8)
       thread_safe (~> 0.1)
     unf (0.1.4)
       unf_ext
     unf_ext (0.0.7.7)
-    warden (1.2.8)
-      rack (>= 2.0.6)
-    web-console (4.0.4)
+    warden (1.2.9)
+      rack (>= 2.0.9)
+    web-console (4.1.0)
       actionview (>= 6.0.0)
       activemodel (>= 6.0.0)
       bindex (>= 0.4.0)
       railties (>= 6.0.0)
-    webpacker (5.1.1)
+    webpacker (5.2.1)
```

Comment:
### [webpacker](https://github.com/rails/webpacker)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 5.1.1 → 5.2.1

[_[change-log](https://github.com/rails/webpacker/blob/v5.2.1/CHANGELOG.md), [source-code](https://github.com/rails/webpacker/tree/v5.2.1)_]





<details>
<summary>Commits</summary>

A change of **49** commits. See the full changes on [the compare page](https://github.com/rails/webpacker/compare/v5.1.1...v5.2.1).

These are the first 10 commits:
- (4bbb461) [Update dependencies (#2563)](https://github.com/rails/webpacker/commit/4bbb4610bb50c9627b3ab6818483897a1de6aad4)
- (48a101b) [Bump webpack to ^4.43.0 (#2550)](https://github.com/rails/webpacker/commit/48a101b8c4ab85dcf8679b6537f555edbd59b59c)
- (0811696) [Update README for optional TS type checking (#2548)](https://github.com/rails/webpacker/commit/08116964dd73a20bad03234d81f6aca2b5d38720)
- (74baab6) [remove default setting from WebpackAssetsManifest (#2565)](https://github.com/rails/webpacker/commit/74baab665f5fda40f108d1a5352dd2a6b23adc83)
- (43f0a9a) [Fix image_pack_tag with srcset does not compute relative pat…](https://github.com/rails/webpacker/commit/43f0a9aa2aca930ed4f6ddbc09080bb828519701)
- (1210f3c) [Merge resolved_paths and watched_paths into additional_paths…](https://github.com/rails/webpacker/commit/1210f3c9ca3f1c85598c8d84483141edc2a66cf4)
- (23726ab) [Update yarn syntax for adding dev package (#2567)](https://github.com/rails/webpacker/commit/23726abb345cc2b5ebdcc3132cb7f5adb35240fb)
- (a7b5197) [De-symlink paths in babel loader to match Webpack (#2329)](https://github.com/rails/webpacker/commit/a7b519727efa01241809701044b4f21d71fe2d65)
- (50ac0e7) [Update README to fix links to typescript.md (#2568)](https://github.com/rails/webpacker/commit/50ac0e75c59b0985f8c93de989627b31a7678663)
- (9e562b4) [Allow configuration of the WEBPACKER_CONFIG via env (#2561)](https://github.com/rails/webpacker/commit/9e562b4c8527ab10cf854c106247590d7eaa49c8)


</details>